README.md

```diff
@@ -52,7 +53,21 @@ In [Azure](https://portal.azure.com/) create an Application Insights resource wh
 
 Add the Instrumentation key from this account into your `local.settings.json` file under the key “APPINSIGHTS_INSTRUMENTATIONKEY”.
 
-## 6. Run the Azure Functions Locally with Visual Studio Code
+## 6. Create `Settings.json`
+Create a `Settings.json` file in the GitHub.Collectors.Functions project.
+
+Find the 'Settings.template.json' file and copy its contents into your new `Settings.json` file.
+
+Add your GitHub account identity under the key “Identity”.
```

Comment:
I really wonder where all you guys get these typeset quotes from :) What's wrong with plain `"`?

Reply:
Haha, it's the Python programmer in me. The double quotes are more consistent with their formatting in Markdown though.